metecho/api/jobs.py

```diff
@@ -280,10 +279,11 @@ def create_repository(
                 raise Exception("Failed to push files to GitHub repository")
 
         # Copy branch protection rules from the template repo
-        if tpl_repo:
-            copy_branch_protection(
-                source=tpl_repo.branch(branch_name), target=repo.branch(branch_name)
-            )
+        # See copy_branch_protection() for why we don't use this currently
+        # if tpl_repo:
+        #     copy_branch_protection(
+        #         source=tpl_repo.branch(branch_name), target=repo.branch(branch_name)
+        #     )
```

Comment:
I really dislike commented code, but as this seems to be an issue with GitHub App perms I'd like to keep this around. I'm considering a more concise repro and logging the issue with them.